cli/connect/language_test.go

```diff
@@ -81,29 +81,6 @@ func (evaler *inputEvaler) Eval(fun string,
 	return nil, errors.New("any error")
 }
 
-func TestChangeLanguage_requestInputs(t *testing.T) {
-	expectedFun := "return require('console').eval(...)\n"
-	expectedOpts := connector.RequestOpts{}
-	cases := []struct {
-		lang Language
-		arg  string
-	}{
-		{DefaultLanguage, "\\set language lua"},
-		{LuaLanguage, "\\set language lua"},
-		{SQLLanguage, "\\set language sql"},
-	}
-
-	for _, c := range cases {
-		t.Run(c.lang.String(), func(t *testing.T) {
-			evaler := &inputEvaler{}
-			ChangeLanguage(evaler, c.lang)
-			assert.Equal(t, expectedFun, evaler.fun)
-			assert.Equal(t, c.arg, evaler.args[0].(string))
-			assert.Equal(t, expectedOpts, evaler.opts)
-		})
-	}
-}
```

Comment:
It seems to me that it would be better to update the test according to the new implementation.

Reply:
Restored and updated this test.